### PR TITLE
[Task CS-71] Create get candidate endpoint

### PR DESCRIPTION
**Pull request by twrdyyy**

### CODE REVIEW

File: rest/routes/candidate.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    """
+    logging.info("Getting candidate data")
+
+    role, response = handle_request_token(request)
+
+    if role is None:
+        logging.warning("Role is not specified!")
+        return response
+
+    if role != 'editor' or role != 'admin':
```
Comment (jmysliv) [inline rest/routes/candidate.py:28]: Why only editor has access to candidate data?

File: rest/routes/candidate.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        logging.warning("Role is not an editor!")
+        return create_response(
+            {"error": "Tylko edytor moze otrzymac informacje o kandydacie."},
+            403,
+            '*'
+        )
+
+    data = dict()
+    candidate = Candidate.filter_by(pesel=candidate_pesel)
```
Comment (jmysliv) [inline rest/routes/candidate.py:37]: Use `get()` instead of `filter_by()` when you query by primary key.

**Review by jmysliv — CHANGES_REQUESTED** [review]
Where are tests for that endpoint?

**Comment by twrdyyy** [comment]
> Where are tests for that endpoint?

Gonna add tomorrow, forgot about it

File: rest/routes/candidate.py (hunk 1 above)
Comment (twrdyyy) [reply rest/routes/candidate.py:28]: who shall have access to candidate data then?

Comment (twrdyyy) [reply rest/routes/candidate.py:28]: imho it is sensitive data and not all the users should have access to it

File: rest/routes/candidate.py (hunk 2 above)
Comment (twrdyyy) [reply rest/routes/candidate.py:37]: > Where are tests for that endpoint?

We cannot add tests as we do not have generator for a single candidate, we will include tests in later version <3

**Review by jmysliv — APPROVED** [review]
lgtm

**Review by skusnierz — APPROVED** [review]

**Pull request merged**